src/requestManager.cpp

```diff
@@ -315,13 +315,13 @@ void RequestBlock(CNode* pfrom, CInv obj)
   // time the block arrives, the header chain leading up to it is already validated. Not
   // doing this will result in the received block being rejected as an orphan in case it is
   // not a direct successor.
+  if (IsChainNearlySyncd()) // only download headers if we're not doing IBD.  The IBD process will take care of it's own headers.
   {
-    LogPrint("net", "getheaders (%d) %s to peer=%d\n", pindexBestHeader->nHeight, obj.hash.ToString(), pfrom->id);  
+    LogPrint("thin", "getheaders (%d) %s to peer=%d\n", pindexBestHeader->nHeight, obj.hash.ToString(), pfrom->id);  
```

Comment:
true, i was trying to see the messages during debugging without having
to put full net debug on...i can change it back.

On 04/09/2016 3:36 AM, gandrewstone wrote:
>
> In src/requestManager.cpp
> <https://github.com/BitcoinUnlimited/BitcoinUnlimited/pull/79#discussion_r77450214>:
>
> >    {
> > -    LogPrint("net", "getheaders (%d) %s to peer=%d\n", pindexBestHeader->nHeight, obj.hash.ToString(), pfrom->id);  
> > +    LogPrint("thin", "getheaders (%d) %s to peer=%d\n", pindexBestHeader->nHeight, obj.hash.ToString(), pfrom->id);  
>
> this GETHEADERS req is not thin block specific...
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/BitcoinUnlimited/BitcoinUnlimited/pull/79/files/efaaa2474cc017ebc1c84fa3748b50c293bff5c7#r77450214>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AMRF0MvQJYV7-a8r0Ds-Qnyy-BoJ0vcoks5qmp8mgaJpZM4JokbP>.
>